src/components/dev-hub/controlled-tooltip.js

```diff
@@ -4,7 +4,10 @@ import { css } from '@emotion/react';
 import Popover from 'react-tiny-popover';
 import { animationSpeed, layer, size } from './theme';
 
-const ControlledTooltipWrapper = styled('span')`
```

Comment:
<img width="201" alt="Screenshot 2021-05-17 at 23 43 28" src="https://user-images.githubusercontent.com/79590294/118554368-06986880-b76a-11eb-93e0-f80fe8b6fe81.png">
Some components are using `span`  for CSS nesting and add margins let's move this logic here.